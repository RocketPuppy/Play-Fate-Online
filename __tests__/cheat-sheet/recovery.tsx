import * as React from "react";
import * as renderer from "react-test-renderer";
import Recovery from "../../src/cheat-sheet/recovery";

test("renders all the pieces", () => {
  const render = renderer.create(<Recovery />);

  expect(render.toJSON()).toMatchSnapshot();
});
