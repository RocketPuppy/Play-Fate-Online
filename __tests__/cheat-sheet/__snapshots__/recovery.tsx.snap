// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Recovery
  </h2>
  <dl>
    <dt>
      Mild
    </dt>
    <dd>
      Overcome Fair, one whole scene
    </dd>
    <dt>
      Moderate
    </dt>
    <dd>
      Overcome Great, one whole session
    </dd>
    <dt>
      Severe
    </dt>
    <dd>
      overcome Fantastic, one whole scenario
    </dd>
  </dl>
</div>
`;
